src/app/core/services/cluster.service.spec.ts

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {HttpClientTestingModule, HttpTestingController} from '@angular/common/http/testing';
+import {TestBed} from '@angular/core/testing';
+import {lastValueFrom} from 'rxjs';
+import {MatDialogModule} from '@angular/material/dialog';
+import {MatSnackBarModule} from '@angular/material/snack-bar';
+import {RouterTestingModule} from '@angular/router/testing';
+
+import {AppConfigService} from '@app/config.service';
+
+import {environment} from '@environments/environment';
+
+import {fakeProject} from '@test/data/project';
+import {RequestType} from '@shared/enums/request-type';
+import {machineDeploymentsFake} from '@test/data/node';
+import {fakeDigitaloceanCluster} from '@test/data/cluster';
+import {AppConfigMockService} from '@test/services/app-config-mock';
+
+import {ClusterService} from './cluster';
+
+describe('ClusterService', () => {
```

Comment:
File name should be `cluster.spec.ts` just to be consistent with other spec file names in `core/services`